config-dev.json

```diff
@@ -106,11 +106,11 @@
     "contracts/core/loan/v1/interfaces/ILoanFDT.sol": {
       "urls": ["contracts/core/loan/v1/interfaces/ILoanFDT.sol"]
     },
-    "contracts/core/loan/v1/Loan.sol": {
-      "urls": ["contracts/core/loan/v1/Loan.sol"]
+    "contracts/core/loan/v1/LoanV1.sol": {
```

Comment:
I'm going to review this and ignore the contract name change because we need to have a discussion about it.